add bin

ch1y0q opened the pull request.

ch1y0q committed:
add bin

FM7077 commented:
`dpkg-buildpackage: info: source package xware-desktop
dpkg-buildpackage: info: source version 1:0.13.20141115
dpkg-buildpackage: info: source distribution trusty
dpkg-buildpackage: info: source changed by Xinkai Chen <yeled.nova@gmail.com>
dpkg-buildpackage: info: host architecture amd64
 dpkg-source --before-build XwareDesktop-master
 debian/rules clean
dh clean
   dh_auto_clean
	make -j1 clean
make[1]: Entering directory '/home/fm7077/Downloads/XwareDesktop-master'
# arch packaging
rm -rf pkg
# deb packaging
rm -rf debian/xware-desktop
rm -rf debian/files
rm -rf debian/*.debhelper
rm -rf debian/*.debhelper.log
rm -rf debian/*.substvars
# build
rm -rf build
rm -rf preparedXware
# in-place
find src/frontend -name "ui_*.py" -print0 | xargs -0 rm -f
find src/frontend -name "*_rc.py" -print0 | xargs -0 rm -f
find src -name "__pycache__" -print0 | xargs -0 rm -rf
find src/frontend -name "*.js" -print0 | xargs -0 rm -f
# extensions
make -C src/frontend/Extensions clean
make[2]: Entering directory '/home/fm7077/Downloads/XwareDesktop-master/src/frontend/Extensions'
rm -f *.so
rm -f *.so.*
rm -f *.dll
rm -f sip/*.sbf
rm -f sip/*.h
rm -f sip/*.cpp
rm -f sip/Makefile
rm -f sip/*.o
rm -f sip/*.so
rm -f sip/*.so.*
rm -f sip/*.exp
rm -f src/Makefile
rm -rf src/build
rm -f src/*.o
make[2]: Leaving directory '/home/fm7077/Downloads/XwareDesktop-master/src/frontend/Extensions'
make[1]: Leaving directory '/home/fm7077/Downloads/XwareDesktop-master'
   dh_clean
	rm -f debian/debhelper-build-stamp
	rm -rf debian/.debhelper/
	rm -f -- debian/xware-desktop.substvars debian/files
	rm -fr -- debian/xware-desktop/ debian/tmp/
	find .  \( \( \
		\( -path .\*/.git -o -path .\*/.svn -o -path .\*/.bzr -o -path .\*/.hg -o -path .\*/CVS -o -path .\*/.pc -o -path .\*/_darcs \) -prune -o -type f -a \
	        \( -name '#*#' -o -name '.*~' -o -name '*~' -o -name DEADJOE \
		 -o -name '*.orig' -o -name '*.rej' -o -name '*.bak' \
		 -o -name '.*.orig' -o -name .*.rej -o -name '.SUMS' \
		 -o -name TAGS -o \( -path '*/.deps/*' -a -name '*.P' \) \
		\) -exec rm -f {} + \) -o \
		\( -type d -a -name autom4te.cache -prune -exec rm -rf {} + \) \)
 dpkg-source -b XwareDesktop-master
dpkg-source: info: using source format '3.0 (native)'
dpkg-source: info: building xware-desktop in xware-desktop_0.13.20141115.tar.xz
dpkg-source: info: building xware-desktop in xware-desktop_0.13.20141115.dsc
 debian/rules build
dh build
   dh_update_autotools_config
   dh_auto_configure
   dh_auto_build
	make -j1
make[1]: Entering directory '/home/fm7077/Downloads/XwareDesktop-master'
mkdir -p build
gcc -Wall -O3 -m32 -o build/etmpatch.so -fPIC -shared -ldl src/etmpatch.c
gcc -Wall -O3 -o build/chmns src/chmns.c
pyuic5 -o src/frontend/legacy/ui_main.py     src/frontend/ui/main.ui
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/lib/python3/dist-packages/PyQt5/uic/pyuic.py", line 26, in <module>
    from PyQt5 import QtCore
ModuleNotFoundError: No module named 'sip'
Makefile:66: recipe for target 'pyqt' failed
make[1]: *** [pyqt] Error 1
make[1]: Leaving directory '/home/fm7077/Downloads/XwareDesktop-master'
dh_auto_build: make -j1 returned exit code 2
debian/rules:7: recipe for target 'build' failed
make: *** [build] Error 2
dpkg-buildpackage: error: debian/rules build subprocess returned exit status 2
`

你好，非常感谢你分享这个软件并写出如此详细的教程。
我的Ubuntu是18.04的
上面是我在编译的时候遇到的错误提示，之前的依赖包已全部安装好，检查时也没再报错，但就在这编译的时候出错了。
在装qtbase5-dev和python3-sip-dev时由于一直报错，在网上找到了这个解决办法[https://blog.csdn.net/T_NULL/article/details/80869396](url)
按这个方法成功装上了以上两个依赖包。
不知道编译错误跟这两个依赖包的安装有没有关系。

再次感谢。

koserLeos commented:
发行版: Ubuntu 18.04.2 LTS 18.04
桌面环境: ubuntu:GNOME/ubuntu
应用程序版本: 408a451b1548864ff5073e60c52e62082d2219cf

补充描述计算机。
可留空。

简述在什么情况下发生了这个问题。
可留空。

======================== 报告 ========================
错误发生在MainThread
```
Traceback (most recent call last):
  File "/usr/bin/xware-desktop", line 282, in <module>
    app = XwareDesktop(sys.argv)
  File "/usr/bin/xware-desktop", line 83, in __init__
    self.sessionService = SessionService(self)
  File "/opt/xware-desktop/frontend/Services/_DBusSessionService.py", line 18, in __init__
    raise RuntimeError("There's a DBus that has the same name.")
RuntimeError: There's a DBus that has the same name.
```
======================== 结束 ========================

TuvokYang commented:
> 发行版: Ubuntu 18.04.2 LTS 18.04
> 桌面环境: ubuntu:GNOME/ubuntu
> 应用程序版本: [408a451](https://github.com/Xinkai/XwareDesktop/commit/408a451b1548864ff5073e60c52e62082d2219cf)
> 
> 补充描述计算机。
> 可留空。
> 
> 简述在什么情况下发生了这个问题。
> 可留空。
> 
> ======================== 报告 ========================
> 错误发生在MainThread
> 
> ```
> Traceback (most recent call last):
>   File "/usr/bin/xware-desktop", line 282, in <module>
>     app = XwareDesktop(sys.argv)
>   File "/usr/bin/xware-desktop", line 83, in __init__
>     self.sessionService = SessionService(self)
>   File "/opt/xware-desktop/frontend/Services/_DBusSessionService.py", line 18, in __init__
>     raise RuntimeError("There's a DBus that has the same name.")
> RuntimeError: There's a DBus that has the same name.
> ```
> 
> ======================== 结束 ========================

你可以使用我修改后的，原作者已经消失很久了

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 160.66kb -> 141.82kb (11.73%)

/src/frontend/ui/icons/scalable/apps/xware-desktop.svg -- 45.47kb -> 30.94kb (31.96%)
/src/frontend/ui/icons/512x512/apps/xware-desktop.png -- 85.95kb -> 82.47kb (4.05%)
/src/frontend/ui/icons/256x256/apps/xware-desktop.png -- 29.24kb -> 28.42kb (2.82%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

ch1y0q committed:
Merge pull request #1 from ch1y0q/imgbot

[ImgBot] Optimize images
